src/commands/setup.rs

```diff
@@ -46,18 +48,43 @@ impl Setup {
             set_sysctl_value(IPV4_FORWARD, "1")?;
         }
 
-        //Configure Bridge and veth_pairs
-        let response = network::core::Core::bridge_per_podman_network(
-            &network_options,
-            &self.network_namespace_path,
-        )?;
+        let mut response: HashMap<String, types::StatusBlock> = HashMap::new();
 
         // Perform per-network setup
         for (net_name, network) in network_options.network_info {
-            debug!("Setting up network {} firewall rules", net_name);
+            debug!(
+                "Setting up network {} with driver {}",
+                net_name, network.driver
+            );
 
-            // Setup basic firewall rules for each network.
-            firewall_driver.setup_network(network)?;
+            match network.driver.as_str() {
```

Comment:
I'd love to get this behind an interface eventually similar to what I'm doing with the firewall driver